modules/plottinggl/glsl/pcp_lines.frag

```diff
@@ -27,11 +30,15 @@ void main() {
         if(subtractiveBelnding == 1){
             res.rgb = 1-res.rgb;
         }
-        if (filtered == 1) 
+        if (filtered == 1) {
              res.xyz = mix(res.xyz, filterColor.xyz, filterIntensity);
+			 res.a = filteredAlpha;
+		}
         if (additiveBlend) {
             res.a *= alpha * pow(lfalloffAlpha, falllofPower);
         }
+		if (hovered == 1)
+			res.xyz = texture(tfSelection, vec2(ltexCoord.y,0.5f)).xyz;
```

Comment:
Formating in this shader is a bit of, not sure if clang-format runs on .frag files. But atleast change tabs to spaces